include/alpaka/core/BoostPredef.hpp

```diff
@@ -12,18 +13,16 @@
 #endif
 
 //---------------------------------------HIP-----------------------------------
-// __HIPCC__ is defined by hipcc (if either __CUDACC__ is defined)
+// __HIP__ is defined by both hip-clang and vanilla clang in HIP mode.
 // https://github.com/ROCm-Developer-Tools/HIP/blob/master/docs/markdown/hip_porting_guide.md#compiler-defines-summary
 #if !defined(BOOST_LANG_HIP)
-#    if defined(__HIPCC__) && (defined(__CUDACC__) || defined(__HIP__))
-#        include <hip/hip_runtime.h>
```

Comment:
This was actually the reason for the CI failures. `__forceinline__` is not a HIP keyword but defined somewhere in the header files included by `hip_runtime.h`.